cmd/provisioner-localpv/app/config.go

```diff
@@ -227,3 +227,10 @@ func GetNodeHostname(n *v1.Node) string {
 	}
 	return hostname
 }
+
+// GetTaints extracts the Taints from the Spec on the node
+// If Taints are empty, it just returns empty structure of corev1.Taints
+func GetTaints(n *v1.Node) []v1.Taint {
+	//var taints []v1.Taint
```

Comment:
n_: remove this unused line. Also can this method be moved to k8s/node builder package.

Reply:
But this function doesn't obey BuilderPattern, and might look different from others.